egs/vcc2020/tts1_en_de/run.sh

```diff
@@ -0,0 +1,310 @@
+#!/bin/bash
+
+# Copyright 2020 Nagoya University (Wen-Chin Huang)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+backend=pytorch
+stage=-1
+stop_stage=100
+ngpu=1       # number of gpu in training
+nj=64        # numebr of parallel jobs
+dumpdir=dump # directory to dump full features
+verbose=0    # verbose option (if set > 1, get more log)
+seed=1       # random seed number
+resume=""    # the snapshot path to resume (if set empty, no effect)
+
+# feature extraction related
+fs=16000      # sampling frequency
+fmax=""       # maximum frequency
+fmin=""       # minimum frequency
+n_mels=80     # number of mel basis
+n_fft=1024    # number of fft points
+n_shift=256   # number of shift points
+win_length="" # window length
+
+# silence part trimming related
+do_trimming=true
+trim_threshold=60 # (in decibels)
+trim_win_length=1024
+trim_shift_length=256
+trim_min_silence=0.01
+
+trans_type=char
+
+# config files
+train_config=conf/train_pytorch_transformer+spkemb.yaml
+decode_config=conf/decode.yaml
+
+# decoding related
+model=model.loss.best
+griffin_lim_iters=64  # the number of iterations of Griffin-Lim
+
+# specify the downloaded database directories
+db_root=../m_ailabs/tts1/downloads
+
+# exp tag
+tag="" # tag for managing experiments.
+
+. utils/parse_options.sh || exit 1;
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+dev_set="dev"
+eval_set="eval"
+
+if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
+    echo "stage -1: Data Download"
+    echo "Please download the M-AILABS datasets in the recipe."
+fi
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    echo "stage 0: Data preparation"
+
+    if [ ! -e ${db_root} ]; then
+        echo "${db_root} not found."
+        echo "cd ${db_root}; ./run.sh --stop_stage -1; cd -"
```

Comment:
If we perform the above modification, you can just do like that
```suggestion
        echo "./run.sh --stage -1"
```
Or you can delete this check.